demes/demes.py

```diff
@@ -92,7 +92,10 @@ def isclose(a: Optional[Number], b: Optional[Number]) -> bool:
         return (a,) == (b,)
     else:
         return math.isclose(
-            typing.cast(typing.SupportsFloat, a), typing.cast(typing.SupportsFloat, b)
+            typing.cast(typing.SupportsFloat, a),
+            typing.cast(typing.SupportsFloat, b),
+            rel_tol=1e-9,
```

Comment:
Always tricky deciding what these tolerances should be - why have you changed them here? I think it'd be better to add the parameters to the function using the same defaults as the builtin, and then change the tolerances in the specific cases where you need it. There's nasty gotchas down the line otherwise.

Reply:
[`math.isclose()`](https://docs.python.org/3/library/math.html#math.isclose) has defaults of `rel_tol=1e-9, abs_tol=0`. The rel_tol is fine, I just added that for clarity. I changed from the default `abs_tol=0` to `abs_tol=1e-12`, because I think we generally want `isclose(0, epsilon)` to be true.